### ISSUE TITLE
Add a QR Code to the About Screen

### ISSUE THREAD

**Issue by MilkManzJourDaddy** [TeamNewPipe/NewPipe#4064]
Add a QR Code in the `About` info' screen, for sharing and on-boarding others to get NewPipe.  This should link to a web-page maintained by the developers, and the page should have a brief explanation of the various repo's, and embedded links to those.

**Comment by avently** [comment]
Isn't it easier to just google with "NewPipe"? The first link goes to NewPipe website with all information you ask.

**Comment by MilkManzJourDaddy** [comment]
Be certain any such About/Info' pane content linked via a QR Code, e.g. **both** F-Droid repo's, have the fingerprint in the link.

**Comment by MilkManzJourDaddy** [comment]
Yeah, *okay*...  Google®️ and not Searx or DDG, et cetera...  Should we also not use NewPipe, but use whatever app' and unvetted **Client-Side** JavaScript laden website YouTube®️ wants us to use?  Plenty of app's on F-Droid contain QR Codes, **for sharing with others**, *OBVIOUSLY*.  It's much easier for you to **NOT WRITE ANY PR/MR**, thanks.
